src/unity/python/turicreate/data_structures/sframe.py

```diff
@@ -4866,8 +4866,11 @@ def unpack(self, column_name, column_name_prefix=None, column_types=None,
 
         Parameters
         ----------
-        column_name : str
-            Name of the unpacked column
+        column_name : str, optional
+            If provided, it unpacks the column with the given name. If not provided
```

Comment:
Nit: this reads a little strangely to me, since the antecedent of "it" isn't immediately clear. Consider just augmenting the original comment, e.g. "Name of the unpacked column, if provided. If not provided..." (or something like that)